easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb

```diff
@@ -0,0 +1,326 @@
+    }),
+    ('biom-format', '2.1.15', {
+        'modulename': 'biom',
+        'checksums': ['3bda2096e663dc1cb6f90f51b394da0838b9be5164a44370c134ce5b3b2a4dd3'],
+    }),
+    ('pylatexenc', '2.10', {
+        'checksums': ['3dd8fd84eb46dc30bee1e23eaab8d8fb5a7f507347b23e5f38ad9675c84f40d3'],
+    }),
+    ('bibtexparser', '1.4.0', {
+        'checksums': ['ca7ce2bc34e7c48a678dd49416429bb567441f26dbb13b3609082d8cd109ace6'],
+    }),
+    ('deblur', '1.1.0', {
+        'checksums': ['78ca2c9946ed99c0d49352e92b63083ae10d04734af7682baddb2c31966c1674'],
+    }),
+    ('emperor', '1.0.4', {
+        'checksums': ['8b57d6ee3709b62b4dfdd8e1f815dd88e5fee3a05947d72fa32191a16ac08854'],
+    }),
+    ('fastcluster', '1.2.6', {
+        'checksums': ['aab886efa7b6bba7ac124f4498153d053e5a08b822d2254926b7206cdf5a8aa6'],
+    }),
+    ('gneiss', '0.4.6', {
+        'preinstallopts': "sed -i '/nose/d' setup.py && ",
+        'checksums': ['5ebf32148909cf74475e45682042c68224f002c1cbf723a5c96014a30f1cf323'],
+    }),
+    ('toolz', '0.12.0', {
+        'checksums': ['88c570861c440ee3f2f6037c4654613228ff40c93a6c25e0eba70d17282c6194'],
+    }),
+    ('ijson', '3.2.3', {
+        'checksums': ['10294e9bf89cb713da05bc4790bdff616610432db561964827074898e174f917'],
+    }),
+    ('iow', '1.0.6', {
+        'modulename': 'bp',
+        'preinstallopts': """sed -i "/'nose /d" setup.py && """,
+        'checksums': ['ac4f579a0881f1c827f75033a304341ae91d3a527cc6aed3fea53749f1b9951c'],
+    }),
+    ('atpublic', '4.0', {
+        'modulename': 'public',
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
+        'checksums': ['80057c55641253b86dcb68b524f82328172371b6547d4c7462a9127fbfbbabfc'],
+    }),
+    ('flufl-lock', '8.0.1', {
+        'modulename': 'flufl.lock',
+        'source_tmpl': 'flufl_lock-%(version)s-py3-none-any.whl',
+        'checksums': ['a3df854d76173d59813fdcba91671234b59e2a14db3390793745c77a7bb92d9d'],
+    }),
+    ('astor', '0.8.1', {
+        'checksums': ['6a6effda93f4e1ce9f618779b2dd1d9d84f1e32812c23a29b3fff6fd7f63fa5e'],
+    }),
+    ('interface_meta', '1.3.0', {
+        'checksums': ['8a4493f8bdb73fb9655dcd5115bc897e207319e36c8835f39c516a2d7e9d79a1'],
+    }),
+    ('wrapt', '1.14.1', {
+        'checksums': ['380a85cf89e0e69b7cfbe2ea9f765f004ff419f34194018a6827ac0e3edfed4d'],
+    }),
+    ('formulaic', '0.4.0', {
+        'checksums': ['087950518c53a2491deb52b8ddd66833f953882e517a42e476c007cc8f6892c5'],
+    }),
+    ('altair', '5.0.1', {
+        'preinstallopts': """sed -i -e 's/^build-backend = .*/build-backend = "setuptools.build_meta"/g' -e 's/^requires = .*/requires = ["setuptools"]/g' -e 's/^license-files = .*//g' -e 's/dynamic = \["version"\]/version = "%(version)s"/g' pyproject.toml &&  schemadir=%(installdir)s/lib/python%(pyshortver)s/site-packages/altair/vegalite/v5/schema/ &&  mkdir -p $schemadir && cp altair/vegalite/v5/schema/vega-lite-schema.json $schemadir && """,
```

Comment:
This should just be the same as in the 2023.5.1 version
```
'preinstallopts': _preinstallopts,
```

Reply:
fixed in 5496f24cd8